### PR TITLE
[Bazel] Ign launch

### PR DESCRIPTION
**Pull request by JShep1**
Signed-off-by: John Shepherd <john@openrobotics.org>

The command line utility is currently throwing an error, I'll try to resolve that soon.

### CODE REVIEW

File: src/Manager.cc [hunk 1]
```diff
@@ -757,6 +757,7 @@ void ManagerPrivate::LoadPlugin(const tinyxml2::XMLElement *_elem)
   ignition::common::SystemPaths systemPaths;
   systemPaths.SetPluginPathEnv("IGN_LAUNCH_PLUGIN_PATH");
   systemPaths.AddPluginPaths(IGNITION_LAUNCH_PLUGIN_INSTALL_PATH);
+  std::cout << "ign launch plugin path " << IGNITION_LAUNCH_PLUGIN_INSTALL_PATH << std::endl;
```
Comment (nkoenig) [inline src/Manager.cc:760]: Remove this line?

File: BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    "ruby_rspec",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+PROJECT_NAME = "ignition-launch"
+PROJECT_MAJOR = 3
+PROJECT_MINOR = 0
+PROJECT_PATCH = 0
+IGN_DESIGNATION = "/tmp"
```
Comment (mjcarroll) [inline BUILD.bazel:32]:
```suggestion
IGN_DESIGNATION = "dome"
```

File: BUILD.bazel [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+CMAKE_INSTALL_PREFIX = "/tmp"
+IGN_DATA_INSTALL_DIR = "/tmp"
+IGN_LAUNCH_PLUGIN_INSTALL_PATH = "./ign_launch"
+library_location = "/usr"
+IGN_LIBRARY_PATH = "/usr"
```
Comment (mjcarroll) [inline BUILD.bazel:37]: In the other projects, we have been doing something like this:

```suggestion
CMAKE_INSTALL_PREFIX = "./ign_launch"
IGN_DATA_INSTALL_DIR = "./ign_launch"
IGN_LAUNCH_PLUGIN_INSTALL_PATH = "./ign_launch"
library_location = "./ign_launch"
IGN_LIBRARY_PATH = "./ign_launch"
```

File: BUILD.bazel [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+    out = "include/ignition/launch/config.hh",
+    cmakelists = ["CMakeLists.txt"],
+    defines = [
+        "PROJECT_VERSION_MAJOR=%d" % (PROJECT_MAJOR),
+        "PROJECT_MAJOR_VERSION=%d" % (PROJECT_MAJOR),
+        "PROJECT_MINOR_VERSION=%d" % (PROJECT_MINOR),
+        "PROJECT_PATCH_VERSION=%d" % (PROJECT_PATCH),
+        "PROJECT_VERSION_PATCH=%d" % (PROJECT_PATCH),
+        "PROJECT_VERSION_MAJOR=%d" % (PROJECT_MAJOR),
+        "PROJECT_VERSION_MINOR=%d" % (PROJECT_MINOR),
```
Comment (mjcarroll) [inline BUILD.bazel:52]: PROJECT_VERSION_MAJOR is duplicated.  Also do we use both orderings?

File: src/Manager.cc (hunk 1 above)
Comment (mjcarroll) [reply src/Manager.cc:766]: Bump.

File: test/BUILD.bazel [hunk 5]
```diff
@@ -0,0 +1,37 @@
+load(
+    "//ign_bazel:cmake_configure_file.bzl",
+    "cmake_configure_file",
+)
+
+BINARY_DIR = "/tmp"
+SOURCE_DIR = "/tmp"
+CMAKE_BINARY_DIR = "/tmp"
+IGNITION_TOOLS_BINARY_DIRS = "/tmp"
```
Comment (mjcarroll) [inline test/BUILD.bazel:9]: Same path comments as above, I don't think these should be /tmp.

**Pull request merged**